setup.py

```diff
@@ -16,5 +16,5 @@
         'Operating System :: OS Independent',
         'Programming Language :: Python'
```

Comment:
I would think `Programming Language :: Python` is generic already.
But if there is good reason to explicitly define `Programming Language :: Python :: 2` and `Programming Language :: Python :: 3` (or even minor versions?) I have no problem adding them.